hwy/tests/arithmetic_test.cc

```diff
@@ -143,16 +143,24 @@ HWY_NOINLINE void TestAllSaturatingArithmetic() {
   const ForPartialVectors<TestUnsignedSaturatingArithmetic> test_unsigned;
   test_unsigned(uint8_t());
   test_unsigned(uint16_t());
+  test_unsigned(uint32_t());
+  test_unsigned(uint64_t());
```

Comment:
Now that all types are supported, we could simplify this: remove the test_* variables and replace with ForUnsignedTypes(ForPartialVectors<TestUnsignedSaturatingArithmetic>());

And for U+I below it would be ForIntegerTypes. Would you like to make this change? If not, I can also do it after merge. (Bonus: this would safely handle HWY_HAVE_INTEGER64==0, though there is currently no target that lacks u/i64.)